micmac_li3ds/import_autocal.py

```diff
@@ -255,4 +258,4 @@ def get_or_create_transfotree(self, node, transfos):
             'sensor_connections': False,
             'transfos': [t['id'] for t in transfos],
         }
-        return self.api.get_or_create_log('transfotree', transfotree)
+        return self.api.get_or_create('transfotree', transfotree)
```

Comment:
What's the reason for not using `api.get_or_create_transfotree` in `import_autocal`?